tests/swift/tests/Sources/SwiftFlatBuffers/fuzzer_generated.swift

```diff
@@ -4,29 +4,31 @@
 
 import FlatBuffers
 
-public enum Color: UInt8, Enum, Verifiable {
+public struct Color: OptionSet, Enum, Verifiable {
```

Comment:
I wonder what's the difference between keeping it as an enum and an OptionSet.

Reply:
I don't think `OptionSet` _can_ be an `enum`.  The magic seems to require an explicit `rawValue` declaration.  Or do you mean difference between keeping it as an enum as it is today, vs. a struct OptionSet?  The syntax on the swift side becomes nicer... but I'm not 100% sure if code that works with the enum case will work with the struct/OptionSet generation though, so it might be a breaking change.  (But IMO a correct one?)